etl/pypasar/db/sql/postgres/provider/provider.sql

```diff
@@ -0,0 +1,45 @@
+-- *******************************************************************
+-- NAME: provider.sql
+-- DESC: Final table - provider
+-- *******************************************************************
+-- CHANGE LOG:
+-- DATE        VERS  INITIAL  CHANGE DESCRIPTION
+-- ----------  ----  -------  ----------------------------------------
+-- 2024-09-07  1.00           Initial create
+-- 2024-09-07  2.00           Insert data into provider table from staging view
+-- 2024-09-07 3.00            Updated the schema name
+-- *******************************************************************
+
+
+INSERT INTO {OMOP_SCHEMA}.provider
+(
+    provider_id,
+    provider_name,
+    npi,
+    dea,
+    specialty_concept_id,
+    care_site_id,
```

Comment:
Consider joining with the **care_site** table to find the corresponding `care_site_id`, or map SGH to 38004515 (as providers are expected to be from SGH).

Reply:
The SGH value isn't added in the spec https://sidata.plus/PASAR_ETL_Spec/provider.html.
I will validate and add the care_site_id. Thanks

Reply:
Oh, I see. Based on the meeting discussion, we could also consider joining with source_to_concept_map for the `specialty_concept_id` field.

Reply:
Thanks for that!